dnf5/main.cpp

```diff
@@ -246,6 +246,12 @@ void RootCommand::set_argument_parser() {
                 throw libdnf5::cli::ArgumentParserError(
                     M_("{}: Badly formatted argument value \"{}\""), std::string{"setopt"}, std::string(value));
             }
+
+            if (std::string(value) == std::string("installonly_limit=1"))
+                throw libdnf5::cli::ArgumentParserInvalidValueError(
+                    M_("Config error: Error parsing --setopt with key 'installonly_limit', "
```

Comment:
I recommend saying "Configuration" rather than "Config.  And is this actually a parse error?  It parsed, it's just a prohibited value.